.eslintignore

```diff
@@ -0,0 +1 @@
+!.eslintrc.js
```

Comment:
Curious - why is this ignored?

Reply:
It's negated to allow ESLint to run against the config file itself, it doesn't by default. I want to make sure the config file itself adheres to the ruleset.

Reply:
Oh I see, neat